packages/blockchain-extension/extension/commands/startFabricRuntime.ts

```diff
@@ -57,6 +58,8 @@ export async function startFabricRuntime(registryEntry?: FabricEnvironmentRegist
             outputAdapter.log(LogType.ERROR, `Failed to start ${runtime.getName()}: ${error.message}`, `Failed to start ${runtime.getName()}: ${error.toString()}`);
         }
 
+        await ExtensionUtil.sleep(1000);
```

Comment:
why do you need to sleep here?

Reply:
Need this to make sure the Fabric is started before connected.